tsconfig.base.json

```diff
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "strict": false,
+        "noImplicitAny": true,
+        "noImplicitReturns": true,
+        "noFallthroughCasesInSwitch": true,
+        "noUnusedLocals": false,
+        "noUnusedParameters": true,
+
+        "jsx": "react",
+        "sourceMap": true,
+        // TODO: add lib
```

Comment:
I need to verify if `lib` is necessary by using TS in other packages. I'll do that shortly and update this PR in the end